rhcs_edits.txt

```diff
@@ -7,8 +7,8 @@ ceph_docker_image: "rhceph/rhceph-5-rhel8"
 ceph_docker_image_tag: "latest"
 ceph_docker_registry: "registry.redhat.io"
 ceph_docker_registry_auth: true
-node_exporter_container_image: registry.redhat.io/openshift4/ose-prometheus-node-exporter:v4.1
+node_exporter_container_image: registry.redhat.io/openshift4/ose-prometheus-node-exporter:v4.6
 grafana_container_image: registry.redhat.io/rhceph/rhceph-5-dashboard-rhel8:5
-prometheus_container_image: registry.redhat.io/openshift4/ose-prometheus:4.1
-alertmanager_container_image: registry.redhat.io/openshift4/ose-prometheus-alertmanager:4.1
+prometheus_container_image: registry.redhat.io/openshift4/ose-prometheus:4.6
```

Comment:
@guits Here also Please refer here for the correct version images https://bugzilla.redhat.com/show_bug.cgi?id=1925350#c4